app/services/email_sender/notify_sender.rb

```diff
@@ -0,0 +1,34 @@
+require "notifications/client"
+
+module EmailSender
+  class NotifySender
+    def call(address:, **keyword_args)
+      send(address: address, **keyword_args)
+    end
+
+  private
+
+    def send(address:, **keyword_args)
```

Comment:
What are these `keyword_args` for? I can't see they're used anywhere?

Reply:
so at the moment only used for overriding the default `template_id` from the config - so 
https://github.com/alphagov/email-alert-api/pull/188/files#diff-0cc6c4f93994a3add62321b9d512a6e4R14
which ends up https://github.com/alphagov/email-alert-api/pull/188/files#diff-0cc6c4f93994a3add62321b9d512a6e4R30

Am I using them correctly? (I've not used the double splatted keyword args feature before).

The intention is, however, to be able to pass in anything we might need when we come to extend this so things like personalisation variables, reply to addresses, any reference ids we might want. There's a small chat about it https://github.com/alphagov/email-alert-api/pull/188#pullrequestreview-70129238